src/Core/LoginConfig/DependencyInjection/services.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="Shopware\Core\LoginConfig\ConfigBuilder\LoginConfigService">
+            <argument>%shopware.admin_login%</argument>
+            <argument type="tagged_iterator" tag="shopware.login.config.handler"/>
```

Comment:
The constructor only has one argument or how do these get passed in (two in XML)?

Reply:
This line can be removed